### ISSUE TITLE
[New]: Wiki.js Template Bounty Request

### ISSUE THREAD

**Issue by bamtests** [railwayapp/templates#453]
### Contact Details

bambam on discord

### Template URL

https://railway.app/template/1ASvMI

### What does this template deploy?

This template deploys a [Wiki.js](https://js.wiki/) container image with a Postgres backend.

I created this because I wanted to be able use wiki.js, I wanted to run it on railway because I love using railway, and I wanted to share with anyone else who also may want to run Wiki.js.

### Demo project

n/a

**Comment by ndneighbor** [comment]
Absolutely, I think this would be fantastic to payout for, I will reach out via Discord to discuss the amount.
